src/app/core/services/github.service.ts

```diff
@@ -193,12 +193,13 @@ export class GithubService {
   }
 
   fetchAllLabels(): Observable<Array<{}>> {
-    return from(octokit.issues.listLabelsForRepo({owner: ORG_NAME, repo: REPO, headers: GithubService.IF_NONE_MATCH_EMPTY})).pipe(
-      map(response => {
-        return response['data'];
-      }),
-      catchError(err => throwError('Failed to fetch labels.'))
-    );
+    return from(octokit.issues.listLabelsForRepo({owner: ORG_NAME, repo: REPO, per_page: 100, headers: GithubService.IF_NONE_MATCH_EMPTY}))
```

Comment:
`100` is a magic number here. Let us consider making this a constant instead of using magic numbers in this case.